File: snippets/js/array/empty/empty.js
var array = [];
if (!array.length) console.log('array is empty');  // ≡
array.push(100);  // add an item
if (array.length) console.log('array has items');  // ≡
array.splice(0);  // clear the array
if (!array.length) console.log('array is empty again');

File: snippets/java/array/empty/Empty.java
package array;

public class Empty {

    public static void main(String args[]) {
        int[] array = new int[]{3, 2};
        if(array.length == 0){
          System.out.println("\n Array is empty "); 
        }else{
            System.out.println("\n Array has Items ");
        }
        // clear the array by reassigning to empty array 
        array = new int[0];  
        if(array.length == 0){
          System.out.println("\n Array is empty again "); 
        } 
    }
}
